src/app/features/notification/notification-settings/notification-settings.component.scss

```diff
@@ -0,0 +1,133 @@
+@use "variables/colors";
+@use "../../../../styles/variables/breakpoints";
+
+.full-width {
+  width: 100%;
+}
+
+.notifications-container {
+  padding: 16px;
+}
+
+.notification-types-container {
+  margin-top: 30px;
+}
+
+.browser-notifications-container {
+  margin-top: 32px;
+}
+
+.notification-setting-title {
+  margin-bottom: 10px;
+}
+
+.panel-wrapper {
+  box-shadow: 0px 0px 10px 0px rgba(0, 0, 0, 0.1);
```

Comment:
use mat-elevation for consistency?
`@use "@angular/material/core/style/elevation" as mat-elevation;`